docs/how-to/data-hub/delete.md

```diff
@@ -34,24 +34,23 @@ To delete multiple rows, databases and folders, run:
 
 ## Delete database columns
 
-To delete columns in databases, run:
+To delete columns in databases, specify the name of the column and the database it exists in:
```

Comment:
I like to make instructions a little more explicit:

> To delete columns in databases, run the following command, specifying the name of the column and its parent database:

Reply:
fixed